packages/ui-chakra/src/base/Button/Button.tsx

```diff
@@ -2,9 +2,12 @@ import React from "react";
 
 import { Button as CKButton } from "@chakra-ui/react";
 
-type ButtonVariant = "primary" | "secondary" | "tertiary" | "accent" | "error";
+type ButtonVariant = "primary" | "secondary";
 type ButtonProps = {
   variant?: ButtonVariant;
+  width?: string;
+  marginTop?: string;
+  isLoading?: boolean;
```

Comment:
And since you have updated the `Button.stories.tsx`, have you included the updated props in the stories? as in, have a story that shows the button in loading state?

But of course, `storybook` is just a bonus for now, no need to worry about it.